### PR TITLE
[v2] Only call EnableSizable for normal windows

### LINKED ISSUE #1049: Logic to (dis)allow to resize window differs on Windows vs Linux/Darwin
#####################################################
     **V1 users: If you have a technical issue, please do not open a bug this way!** 
                      Please use the `wails issue` command!
          If you do not do this then the issue may be closed automatically.                      

 NOTE: If your bug is related to Windows, make sure you read 
 the [Windows Developer Guide](https://wails.app/guides/windows/)
#####################################################

**Description**

**To Reproduce**

Simple example code:

```
package main

import (
	"embed"
	"log"

	"github.com/wailsapp/wails/v2/pkg/options/mac"

	"github.com/wailsapp/wails/v2"
	"github.com/wailsapp/wails/v2/pkg/logger"
	"github.com/wailsapp/wails/v2/pkg/options"
	"github.com/wailsapp/wails/v2/pkg/options/windows"
)

//go:embed frontend/src
var assets embed.FS

//go:embed build/appicon.png
var icon []byte

func main() {
	// Create an instance of the app structure
	app := NewApp()

	// Create application with options
	err := wails.Run(&options.App{
		Title:     "Your-POS",
		Width:     720,
		Height:    570,
		MinWidth:  720,
		MinHeight: 570,
		//MaxWidth:          1280,
		//MaxHeight:         740,
		DisableResize:     false,
		Fullscreen:        false,
		Frameless:         true,
		StartHidden:       false,
		HideWindowOnClose: false,
		RGBA:              &options.RGBA{R: 33, G: 37, B: 43, A: 255},
		Assets:            assets,
		LogLevel:          logger.DEBUG,
		OnStartup:         app.startup,
		OnDomReady:        app.domReady,
		OnShutdown:        app.shutdown,
		Bind: []interface{}{
			app,
		},
		// Windows platform specific options
		Windows: &windows.Options{
			WebviewIsTransparent: false,
			WindowIsTranslucent:  false,
			DisableWindowIcon:    false,
		},
		Mac: &mac.Options{
			TitleBar:             mac.TitleBarHiddenInset(),
			WebviewIsTransparent: true,
			WindowIsTranslucent:  true,
			About: &mac.AboutInfo{
				Title:   "Vanilla Template",
				Message: "Part of the Wails projects",
				Icon:    icon,
			},
		},
	})

	if err != nil {
		log.Fatal(err)
	}
}
```

By default the window is resizable, if I am not mistaken.

However, using the example above, I have to enable frameless to allow to resize the window (from the edges, using the mouse).

I think it is a bug (or doc issue :). This line causes it I suppose:

https://github.com/wailsapp/wails/blob/6d9a195830ef9e83452695e243bc0771091ea1f1/v2/internal/frontend/desktop/windows/window.go#L60

```
	if appoptions.Frameless == false && !appoptions.Fullscreen {
		result.EnableMaxButton(!appoptions.DisableResize)
		result.EnableSizable(!appoptions.DisableResize) // Here DisableResize is false, so it becomes true if frameless is false and not in fullcreen
		result.SetMinSize(appoptions.MinWidth, appoptions.MinHeight)
		result.SetMaxSize(appoptions.MaxWidth, appoptions.MaxHeight)
	}
```

**Expected behaviour**
Ability to resize the window by default.

**Screenshots**
NA

**System Details**
```

Wails CLI v2.0.0-beta.27


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:             Windows 10 Pro
Version:        2009 (Build: 19044)
ID:             21H2
Go Version:     go1.17.5
Platform:       windows
Architecture:   amd64

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
WebView2        N/A             Installed       96.0.1054.62
npm             N/A             Installed       8.3.0
*upx            N/A             Available

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/


```

**Additional context**

(https://github.com/wailsapp/wails/blob/6d9a195830ef9e83452695e243bc0771091ea1f1/v2/internal/frontend/desktop/windows/window.go#L60) seems to be the culprit.

Other platforms do not have this logic:

Linux:
https://github.com/wailsapp/wails/blob/6d9a195830ef9e83452695e243bc0771091ea1f1/v2/internal/frontend/desktop/linux/window.go#L248
```
	result.SetResizable(!appoptions.DisableResize)
```

Mac:
https://github.com/wailsapp/wails/blob/6d9a195830ef9e83452695e243bc0771091ea1f1/v2/internal/frontend/desktop/darwin/window.go#L49
```
resizable := bool2Cint(!frontendOptions.DisableResize)
```

- [x] This issue is for Windows and I have read the [Windows Developer Guide](https://wails.app/guides/windows/)

### PR DESCRIPTION
**Pull request by stffabi**
Frameless windows are always not resizable per default and
the resizing for those will be initiated by the frontend see
processMessage.

If EnableSizable is enabled for frameless windows, a small white
titlebar will be shown.

Fix #1049

### CODE REVIEW

File: v2/go.mod [hunk 1]
```diff
@@ -80,3 +80,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/winc v0.0.0-20220112092814-1608a1744133 => github.com/stffabi/winc v0.0.0-20220114143400-0c2341f6e43a
```
Comment (stffabi) [inline v2/go.mod:84]: Will be removed as soon as leaanthony/winc#9 has been merged

Comment (leaanthony) [reply v2/go.mod:84]: Merged 🙏

Comment (stffabi) [reply v2/go.mod]: Thanks @leaanthony I've updated this PR.

Comment (leaanthony) [reply v2/go.mod]: I've just merged a different change to winc. Could you possibly update go.mod to use the very latest? Thanks 🙏

Comment (leaanthony) [reply v2/go.mod]: Both you and @taigrr are updating winc 😀

Comment (stffabi) [reply v2/go.mod]: Yes of course, I've just seen that :smile, give me just a couple of minutes.

Comment (stffabi) [reply v2/go.mod]: Done.

I'll come up with another PR for winc maybe in the next hour, but I think we should move this one forward and already merge this PR.

**Comment by leaanthony** [comment]
I've just merged your PR on winc. Does this need updating again? 😆

**Comment by stffabi** [comment]
I've just updated it to contain leaanthony/winc#12

**Comment by leaanthony** [comment]
Glad to finally get this one over the line :-)

**Comment by stffabi** [comment]
:smile Thanks for merging

**Pull request merged**